src/server.c

```diff
@@ -6151,25 +6175,81 @@ sds genRedisInfoStringCommandStats(sds info, dict *commands) {
     return info;
 }
 
+void addSectionsToDict(dict * section_dict, char ** sections, int len) {
+    for (int i = 0; i < len; i++){
+        sds section = sdsnew(sections[i]);
+        dictAdd(section_dict, section, NULL);
+    }
+}
+
+
+dict * genSectionDict(client * c, const char * source, int * all_sections, int * everything) {
+    char * defSections[] = {"server", "clients", "memory", "persistence", "stats", "replication", "cpu", "modules", "errorstats", "cluster", "keyspace"};
+    char * defSectionsSentinel[] = {"server", "clients", "cpu", "stats"};
+
+
+    dict * section_dict = dictCreate(&stringSetDictType); /* Set to add the subsections to print*/
+
+    if (c == NULL || c->argc == 1) {
+        if (!strcasecmp(source,"sentinel")){
+            addSectionsToDict(section_dict, defSectionsSentinel, sizeof(defSectionsSentinel)/sizeof(*defSectionsSentinel));
+        }
+        else if (!strcasecmp(source,"server")){
+            addSectionsToDict(section_dict, defSections, sizeof(defSections)/sizeof(*defSections));
+        }
+    }
+    else {
```

Comment:
styling (fix in other places too)
```suggestion
    } else {
```

Reply:
Addressed the style.